freebites/product
# 81 post description edits

# Description

Updated general styling for the post page as well as the info modal. Added a display in the top left of the post image showing the posts creator's name and profile picture. Also made the active user's profile picture appear in the new comment posting display and have all commenters profile pictures appearing next to their names.

## Issues

#81 

## Screenshots

<img src="https://github.com/user-attachments/assets/273aabcc-2994-459e-a601-bb18bf09bd43" width=250/>
<img src="https://github.com/user-attachments/assets/f506dae6-9638-4285-b860-c58655b1758f" width=250 />
<img src="https://github.com/user-attachments/assets/878e87e2-4082-49fa-a3b2-c21cd413fa18" width=250/>


## Test

1. tap on any post on the home screen.
2. check that floating display is on the top left of the post image
3. try tapping the (i) button next to Food types and Diet to see the new style
4. tap the bottom bar and try adding a comment, checking that all the other comments have correct profile pictures as well.

## Possible Downsides

We don't think there are any downsides
